src/features/CourtCaseDetails/Sidebar/TriggersAndExceptions.tsx

```diff
@@ -0,0 +1,55 @@
+import { Tabs } from "govuk-react"
+import { useState } from "react"
+import { createUseStyles } from "react-jss"
+import CourtCase from "../../../services/entities/CourtCase"
+import Triggers from "./Triggers"
+import Exceptions from "./Exceptions"
+import type { AnnotatedHearingOutcome } from "@moj-bichard7-developers/bichard7-next-core/build/src/types/AnnotatedHearingOutcome"
+
+interface Props {
+  courtCase: CourtCase
+  aho: AnnotatedHearingOutcome
+}
+
+const useStyles = createUseStyles({
+  pointer: {
+    cursor: "pointer"
+  },
+  sideBarMarginTop: {
+    marginTop: "-41px"
+  }
+})
+
+const TriggersAndExceptions: React.FC<Props> = ({ courtCase, aho }) => {
```

Comment:
We can write it the other way (I've added a comment above) & avoid using `React.FC`.